server/backend/cron/news.go

```diff
@@ -120,7 +122,8 @@ func (c *CronService) parseNewsFeed(source model.NewsSource) error {
 				Src:         source.Source,
 				Link:        item.Link,
 				Image:       enclosureUrl,
-				File:        fileId,
+				FilesID:     null.Int{NullInt64: sql.NullInt64{Int64: int64(file.File), Valid: true}},
```

Comment:
This seems a bit manual, just to create a nullable int. There must be a shorthand for this. See https://pkg.go.dev/github.com/guregu/null#IntFrom

Reply:
Good catch!
I assumed this was just one of the go-quirks
⇒ #229 (and rebasing tomorrow 😉)